### ISSUE TITLE
Error Install new Jetstream with Livewire 

### ISSUE THREAD

**Issue by RitsukiKagerou** [laravel/jetstream#1556]
### Jetstream Version

5.3.4

### Jetstream Stack

Livewire

### Laravel Version

11.36.1

### PHP Version

8.3.15

### Description

Follow the installation steps from the official Laravel Jetstream documentation: https://jetstream.laravel.com/installation.html

![image](https://github.com/user-attachments/assets/ac3fb147-7546-4e2c-87fd-1066859e4b19)

### Steps To Reproduce

```
composer create-project laravel/laravel example-app
cd example-app
composer require laravel/jetstream
```

The error occurs when running:
```
php artisan jetstream:install livewire
```

**Comment by RitsukiKagerou** [comment]
Looks like the problem from vite : https://github.com/vitejs/vite/issues/19018

**Comment by SiddharthPant** [comment]
Yes, esbuild to be exact, its v0.24.1 is breaking vite. Maintainers are working on a fix. Till then temporary workaround is to either downgrade manually esbuild to v0.24.0 so `npm i -D esbuild@0.24.0` or else provide override in `package.json` like this:
```json
{
  "name": "Your app",
  "devDependencies": {
    "vite": "^6.0.4",
  },
  "overrides": {
    "vite": {
      "esbuild": "0.24.0"
    }
  },
}
```
All Laravel new starter kits installs are failing today because of this regression.

**Comment by RitsukiKagerou** [comment]
Solved, vite 6.0.5 with esbuild regression (pin to 0.24.0)
https://github.com/vitejs/vite/releases/tag/v6.0.5
